distribution/macos/Info.plist

```diff
@@ -2,6 +2,17 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>CFBundleURLTypes</key>
+	<array>
+		<dict>
+			<key>CFBundleURLName</key>
+			<string>website.openrct2.openrct2</string>
```

Comment:
> To ensure uniqueness, it is recommended that you use a Java-package style identifier.

I used `website.openrct2.openrct2` because the official domain for the project is `openrct2.website`, and the last `openrct2` refers to the scheme.